src/main/java/me/partlysanestudios/partlysaneskies/HelpCommands.java

```diff
@@ -94,7 +94,7 @@ public static void printHelpMessage() {
                 "\n &6> Open the config: " +
                 "\n    &6> &ePress " + Keyboard.getKeyName(Keybinds.configKey.getKeyCode()) + " or use /pssc" +
                 "\n    &6> &eMost features are turned off by default so to use the mod, you will need to configure the settings" +
-                "\n    &6> &eTo change the keybinding, press Esc, Options, Video Settings, Controls, and scroll down to \"Partly Sane Skies\"." +
```

Comment:
Which idiot put "Esc, Options *video setttings*, controls" in the first place

Reply:
Someone needs to be executed